cs/v1/CS_v1.0.0.yaml

```diff
@@ -570,9 +613,9 @@ components:
           x-stoplight:
             id: xfeizdvobewab
           description: The name of the Vessel given by the Vessel Operator and registered with IMO.
-          example: 'King of the Seas'
+          example: King of the Seas
           maxLength: 50
-          pattern: ^\S(?:.*\S)?$
+          pattern: '^\S(?:.*\S)?$'
```

Comment:
I am not sure about this... If you are adding quotes around the regular expression - I would suspect that and backlash `\` needs to be escaped (by adding a double backlash). This is one of these places where StopLight modifies the spec wrong in my POV
I suggest to either remove the quotes or escape the backlash in the string...

Reply:
There are still quotes here...?